src/Bicep.LangServer/Deploy/ParameterType.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Bicep.LanguageServer.Deploy
+{
+    public enum ParameterType
```

Comment:
nit: Enums are easier to debug if you use strings, e.g.:

export enum ParametersFileUpdateOption {
  Create = "Create",
  None = "None",
  Overwrite = "Overwrite",
  Update = "Update",
}